lms/static/sass/theme-overrides.scss

```diff
@@ -68,7 +68,7 @@ a:not(.btn), a:not(.btn):hover, a:not(.btn):focus, a:visited:not(.btn):hover, a:
       }
     }
   }
-  .main .nav-tab a, .secondary .nav-item a:not(.btn), .secondary .nav-item .toggle-user-dropdown {
+  .main .nav-tab a, .secondary .nav-item a:not(.btn):not(.help-link), .secondary .nav-item .toggle-user-dropdown {
```

Comment:
@pkulkark, instead of adding a specific exclusion for an element like this, wouldn't it be better to add a more specific selector to apply the expected colors? If you do so, we don't have to negate the `help-link` class as the style will automatically get applied thanks to CSS specificity.

Reply:
@lgp171188 We are already doing that [here](https://github.com/open-craft/edx-simple-theme/blob/011f0a9145c55b91981b083aacb9b78402b30c78/lms/static/sass/theme-overrides.scss#L43) (i.e. setting the header link color). But It looks like that's getting overriden by this line as it gets executed later.